CLI/src/main/java/main/Drifty_CLI.java

```diff
@@ -40,6 +40,7 @@ public class Drifty_CLI {
     private static String batchDownloadingFile;
     private static final String MSG_FILE_EXISTS_NO_HISTORY = "\"%s\" exists in \"%s\" folder. It will be renamed to \"%s\".";
     private static final String MSG_FILE_EXISTS_HAS_HISTORY = "You have previously downloaded \"%s\" and it exists in \"%s\" folder.\nDo you want to download it again? ";
+    private static final String URL_LIST_FILE = "drifty_urls.txt";
```

Comment:
@Femi-lawal Please consider changing the filename and the directory of the file to :point_down: 
You can use `Program.get(Program.DRIFTY_PATH)` to get the directory for storing the file. The filename must be `links.yml`. You must use YAML file to store the data in the format :point_down: 
```yml
links: ["...", "..."]
```
You can use HashMaps to arrange the data taken from the user and dump it using SnakeYaml. [This StackOverflow Answer](https://stackoverflow.com/a/24950664/19187952) might help you in this context :smile:.

Reply:
@SaptarshiSarkar12 Hi, I've made the changes